exp/addons/internal/controllers/suite_test.go

```diff
@@ -55,13 +55,13 @@ func TestMain(m *testing.M) {
 			Client:  mgr.GetClient(),
 			Tracker: tracker,
 		}
-		if err = reconciler.SetupWithManager(ctx, mgr, controller.Options{MaxConcurrentReconciles: 1}); err != nil {
+		if err = reconciler.SetupWithManager(ctx, mgr, controller.Options{MaxConcurrentReconciles: 10}); err != nil {
```

Comment:
wondering if this changes could cause flakes 🤔

Reply:
No I don't think so because only the new test I added showed the racy behaviour.